gc/base/GCExtensionsBase.hpp

```diff
@@ -406,7 +407,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	uintptr_t workpacketCount; /**< this value is ONLY set if -Xgcworkpackets is specified - otherwise the workpacket count is determined heuristically */
 	uintptr_t packetListSplit; /**< the number of ways to split packet lists, set by -XXgc:packetListLockSplit=, or determined heuristically based on the number of GC threads */
```

Comment:
let's fix this too: we should not refer to specifics of J9. we should state more generically 'by command line option' (rather than -XXgc:packetListLockSplit=), since potentially other projects could use different command line options